### PR TITLE
feat: allowing to delete k8s crashlooping pods

### PR DESCRIPTION
**Pull request by benjaminch**

### CODE REVIEW

File: src/cmd/kubectl.rs [hunk 1]
```diff
@@ -1017,6 +1017,88 @@ where
...
+where
+    P: AsRef<Path>,
+{
+    let restarted_min = restarted_min_count.unwrap_or_else(|| 10usize);
+    let pods = kubectl_exec_get_pods(kubernetes_config, namespace, selector, envs)?;
+
+    Ok(pods
+        .items
+        .into_iter()
+        .filter(|e| {
```
Comment (MacLikorne) [inline src/cmd/kubectl.rs:1045]: does e stand for pod ? If you yes, I'd suggest to switch to an explicit var

File: src/cmd/kubectl.rs [hunk 2]
```diff
@@ -1017,6 +1017,88 @@ where
...
+                    ));
+                }
+
+                pods.items[0].clone()
+            }
+            Err(e) => return Err(e),
+        };
+
+    kubectl_exec(
+        vec!["delete", "pod", pod_to_be_deleted.metadata.name.as_str()],
```
Comment (MacLikorne) [inline src/cmd/kubectl.rs:1094]: If i remember well you have specify namespace and add an arg for pod name:

delete pod -n <namespace> --name <pod_name>

Comment (benjaminch) [reply src/cmd/kubectl.rs]: Indeed, thanks, --name is not an option, but namespace is required.

File: src/cmd/kubectl.rs (hunk 1 above)
Comment (benjaminch) [reply src/cmd/kubectl.rs]: Yep, ok !

**Review by deimosfr — APPROVED** [review]

**Pull request merged**